.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -19,11 +17,7 @@
 <!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
 <!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
 - [ ] I have read the [**CONTRIBUTING**](https://github.com/markdownlint/markdownlint/blob/master/CONTRIBUTING.md) document.
-- [ ] Used the same coding conventions as the rest of the project
 - [ ] Wrote [good commit messages](https://chris.beams.io/posts/git-commit/)
-- [ ] Commit message starts with `[Fix #issue-number]` if there is a corresponding open GitHub issue
 - [ ] Feature branch is up-to-date with `master`, if not - rebase it
-- [ ] All [related commits squashed together](http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html)
 - [ ] Added tests for all new/changed functionality, including tests for positive and negative scenarios
-- [ ] Added an entry to the [Changelog](https://github.com/markdownlint/markdownlint/blob/master/CHANGELOG.md) if the new code introduces user-observable changes. See [Changelog entry format](https://github.com/markdownlint/markdownlint/blob/master/CONTRIBUTING.md#changelog-entry-format)
```

Comment:
I'm ok asking people to add a changelog entry... fwiw

Reply:
that's fine, we can sort out what it should look like first though. Let's just get this in.